tests/test-cases.js

```diff
@@ -1319,5 +1319,20 @@ var CUSTOM_TESTS = [
 				}
 			});
 		}
-	}
+	},
+	{
+		description: "Errors give correct row numbers with several chunks being processed",
+		expected: '[{"type":"FieldMismatch","code":"TooFewFields","message":"Too few fields: expected 12 fields but parsed 11","row":21},{"type":"FieldMismatch","code":"TooFewFields","message":"Too few fields: expected 12 fields but parsed 10","row":23}]',
```

Comment:
It looks to me like these are referring to line 23 and 25 in the file. Shouldn't it be row 22 and 24?